src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -50,6 +51,9 @@ public class DurableTaskExtension :
         private readonly ConcurrentDictionary<FunctionName, RegisteredFunctionInfo> knownActivities =
             new ConcurrentDictionary<FunctionName, RegisteredFunctionInfo>();
 
+        private readonly ConcurrentDictionary<string, DurableOrchestrationContext> continueAsNewContexts =

```

Comment:
This seems to be unused.

Reply:
Thanks, I _knew_ I forgot to delete something but couldn't find it. :) This is a leftover from an alternative fixed I tried (which didn't work and had to be removed).